wordpress-mobile/WordPress-iOS
# Add captions to stock photos images

This PR adds the support on exporting media to add a custom `caption`. 

This is implemented by Stock Photos to add a caption automatically created with the credits of the photo author to the imported media item.

This is a new functionality in Calypso that we are importing to Mobile:

![photo-credit-caption](https://user-images.githubusercontent.com/9772967/40143145-a02b0bb8-5928-11e8-8a26-9082d0499b7e.gif)

The caption comes from the stock photos api.

![stock_photos_caption](https://user-images.githubusercontent.com/9772967/40143433-6c7d996a-5929-11e8-98a6-555d4810e4db.gif)

@SergioEstevao can I ask you for a review please?

To test:

- Go to the media library.
- Add an image from the free photos library.
- Check that the caption was added.
- Check in Calypso that the caption was synchronized.

cc @iamthomasbishop
